contracts/upgradeable_contracts/InterestReceiver.sol

```diff
@@ -42,8 +39,6 @@ contract InterestReceiver is ERC677Receiver, Initializable, Ownable, Claimable,
     * @param _value amount of transferred tokens
     */
     function onTokenTransfer(address, uint256 _value, bytes) external returns (bool) {
```

Comment:
Since we have some time let's introduce sending tokens to the consensus contract in the xDai chain as part of `onTokenTransfer` action. We can use `relayTokens` functionality for this. The address of the receiver in the xDai chain will be specifed in the constructor and can be changed later through `setReceiverInXDai`. The `withdraw()` operation is not needed in this case.